### DISCUSSION TITLE
Change the naming convention for $id to match file names

### DISCUSSION THREAD

**Discussion by pjohnmeyer** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#117] [Suggestions] [answered]
Some schema loaders support loading schemas from a "prefix":

https://opis.io/json-schema/2.x/php-loader.html#registerprefix-method

Currently, our IDs and our filenames don't match, e.g.:

> **file: .../schema/enums/CompensationType.schema.json**
> `$id: https://opencaptablecoalition.com/schema/enums/compensation_type`

If we make the $id values match the file path, in this case `https://opencaptablecoalition.com/schema/enums/CompensationType.schema.json`, it will enable simpler schema loading.

**Comment by JSv4** [comment]
I'm in. Sounds like a great idea.

**Comment by JSv4** [comment, marked as answer]
Making this change via issue #120
